### DISCUSSION TITLE
invalid response

### DISCUSSION THREAD

**Discussion by peebee44** [subhra74/xdm#950] [General]
When trying to download a video I just get "Xdm can not download this file. The server has returned an invalid response"

**Comment by realmzakaria** [comment]
I am getting that too when I try to download youtube videos.

**Comment by GregorySX** [comment]
Stopped working with youtube videos since around two weeks. They must have changed something, used to be MKV files, now they download as WEBM files with yt-dlp.

**Comment by frank392a** [comment]
Because The new XDM sucks !!!!! big time

**Reply by RobotWillie613** [reply]
Has nothing to do with it being old or new Youtube doesn't download on any of them now.

**Comment by peebee44** [comment]
Is anyone going to fix this????

**Reply by frank392a** [reply]
just use the old one it is still great !!

**Reply by RobotWillie613** [reply]
Nope Youtube downloading doesn't work on the old one either!

**Comment by peebee44** [comment]
Why is there no response from the owner? Is he trying to fix it?

**Comment by frank392a** [comment]
For youtube you can use one of this great apps : 
https://flathub.org/apps/details/com.github.unrud.VideoDownloader
https://flathub.org/apps/details/me.aandrew.ytdownloader
https://flathub.org/apps/details/com.ktechpit.ultimate-media-downloader

**Comment by subhra74** [comment]
Please try latest version 8.0.18, and let me know if this solves the issue. 
https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.18-beta

**Reply by chocourrier** [reply]
Yes, the latest version solves the issue for me. Thanks a lot!
